\documentclass{beamer}
\useoutertheme{infolines}
\usepackage[utf8]{inputenc}
\usepackage{default}
%\usepackage{enumitem}
\usepackage{graphicx}
\usepackage[normalem]{ulem}
\newcommand\redout{\bgroup\markoverwith
{\textcolor{red}{\rule[.5ex]{2pt}{2pt}}}\ULon}
\usepackage{xcolor}
\usepackage{textpos}
\usepackage{tikz}   
\usepackage[round]{natbib}
\bibliographystyle{apalike}
\usepackage[T1]{fontenc}
\usepackage{pdfpages}
\usepackage{hyperref}

\graphicspath{{/home/ejmctavish/Desktop/Talks/}}



%\addtobeamertemplate{frametitle}{}{%
%\begin{textblock*}{100mm}(.8\textwidth,-1cm)
%\includegraphics[height=1cm,width=2cm]{opentreelogogrey}
%\end{textblock*}}

\useinnertheme{rectangles}
\usetheme{Copenhagen}
\setbeamertemplate{footline}[frame number]{}
\setbeamertemplate{footline}{}

\title[*]{Phylogenetic terminology and applications}
\author[*]{Emily Jane McTavish}
\institute[*]{
Life and Environmental Sciences\\
University of California, Merced\\
\texttt{ejmctavish@ucmerced.edu, twitter:snacktavish}\\
}
\date{}
\setbeamertemplate{itemize items}[triangle]

\begin{document}

\begin{frame}
\titlepage
(With thanks to Mark Holder, Paul Lewis, Joe Felsenstein, and David Hillis for slides) 
\end{frame}



\begin{frame}
\begin{center}
 \Large{Phylogenies describe shared ancestry\\
and\\
inform our understanding of evolutionary processes}
\end{center}
\end{frame}


\begin{frame}
\begin{center}
\centerline{\includegraphics[scale=0.4]{tol_big}}
\end{center}
%TODO examples!
\tiny{Image Ethan Hein}
\end{frame}
%My interests are in understanding the tree of life


\begin{frame}
\begin{center}
\centerline{\includegraphics[scale=0.25]{tol_big_example}}
\end{center}
\tiny{Image Ethan Hein}
\end{frame}



%-----comp meth examples------
\setbeamercolor{background canvas}{bg=}
\includepdf[pages=3-14]{/home/ejmctavish/Desktop/old_classes/QSB244/GradPhylo/docs/slides/lec1-IntroStats.pdf}


%-------tree terms------
\setbeamercolor{background canvas}{bg=}
\includepdf[pages=18-23]{/home/ejmctavish/Desktop/old_classes/QSB244/GradPhylo/docs/slides/lec1-IntroStats.pdf}

\setbeamercolor{background canvas}{bg=}
\includepdf[pages=24]{/home/ejmctavish/Desktop/old_classes/QSB244/GradPhylo/docs/slides/lec1-IntroStats.pdf}


\begin{frame}
\begin{centering}
\includegraphics[height=0.8\textheight]{/home/ejmctavish/Desktop/old_classes/QSB244/GradPhylo/docs/slides/poly}
\end{centering}\\
from wikipedia
\end{frame}


\begin{frame}
\begin{centering}
\includegraphics[width=\textwidth]{/home/ejmctavish/Desktop/old_classes/QSB244/GradPhylo/docs/slides/xkcdherp}
\end{centering}\\
https://xkcd.com/867/
\end{frame}

\begin{frame}
\begin{center}
Are dogs a monophyletic group in this tree?\\
Are wolves?\\
\centerline{\includegraphics[width=\textwidth]{dogdomestication}}
\end{center}
%TODO examples!
\end{frame}

\begin{frame}
more terms:
\begin{itemize}
 \item sister taxa: taxa or monophyletic groups which share a most recent common ancestor
 \item outgroup: taxon that is determined \emph{a priori} to be sister to all other taxa in the analysis. Used for rooting tree
\end{itemize}

\end{frame}

\setbeamercolor{background canvas}{bg=}
\includepdf[pages=27-29]{/home/ejmctavish/Desktop/old_classes/QSB244/GradPhylo/docs/slides/lec1-IntroStats.pdf}



%----Disease stuff from Hillis -----

\setbeamercolor{background canvas}{bg=}
\includepdf[pages=13-25]{/home/ejmctavish/Desktop/old_classes/QSB244/Hillis/HillisMolEvol2014.pdf}

\begin{frame}
Changing the rooting of this phylogeny would change inferences!
\end{frame}


%-----------------
\begin{frame}
Phylogenies can reveal suprising patterns
 \includegraphics[width=0.75\textwidth]{/home/ejmctavish/Desktop/old_classes/QSB244/GradPhylo/docs/slides/butterflycut}
\end{frame}

\begin{frame}
Phylogenies can reveal suprising patterns
 \includegraphics[width=\textwidth]{/home/ejmctavish/Desktop/old_classes/QSB244/GradPhylo/docs/slides/butterfly}
 \citep{joron_chromosomal_2011}
\end{frame}



\begin{frame}
What evolutionary processes can drive these patterns?
\begin{itemize}
 \item Convergence
 \item Horizontal gene transfer
 \item Within species variation (Incomplete lineage sorting)
 \item ?
\end{itemize}
\pause
We will discuss how to recognize and (try to) differentiate these processes.
\end{frame}

\begin{frame}
Different data can drive different conclusions
 \includegraphics[width=\textwidth]{/home/ejmctavish/Desktop/old_classes/QSB244/GradPhylo/docs/slides/batspptree}\\
Species relationships between echolocating and nonecholocating bats (after Teeling 2009).
Left: inferences from DNA sequence data.\\
Right: traditional species relationships inferred from morphological characters (and limited sequence data).
\citep{hahn_irrational_2016}
\end{frame}



\begin{frame}
Estimating a tree from character data\\
Tree construction:
\begin{itemize}
 \item strictly algorithmic approaches - use a “recipe” to construct a tree
  \item optimality based approaches - choose a way to “score” a trees and then search for the tree that has the best score.
\end{itemize}

Expressing support for aspects of the tree:
\begin{itemize}
 \item  bootstrapping,
 \item testing competing trees against each other,
 \item posterior probabilities (in Bayesian approaches).
\end{itemize}
\end{frame}


\appendix
\begin{frame}[allowframebreaks]
 \bibliography{/home/ejmctavish/Desktop/old_classes/QSB244/GradPhylo/docs/slides/GradPhylo}
\end{frame}



\end{document}
